app/views/layouts/application.html.erb

```diff
@@ -66,6 +66,10 @@
   </head>
 
   <body class="<%= body_classes %>">
+    <%- if allow_plugins? %>
```

Comment:
I took a quick peek at `build_plugin_html` and it seems to check `allow_plugins?` already, so this `if` statement shouldn't be necessary.

Reply:
Yes I got the same question actually, I saw we are doing it at other places, but I also think we could drop it everywhere, can do it in a separate PR if you want.

Reply:
We probably should! I'll approve this now, though. Thanks!